coderd/coderd.go

```diff
@@ -207,6 +207,13 @@ func New(options *Options) *API {
 			r.Get("/parameters", api.templateVersionParameters)
 			r.Get("/resources", api.templateVersionResources)
 			r.Get("/logs", api.templateVersionLogs)
+			r.Route("/plan", func(r chi.Router) {
```

Comment:
We have mixed naming of `DryRun` and `plan` with this change. It'd be best to keep it consistent. I like `plan` better, but chefs choice.